src/config/mekanism.cfg

```diff
@@ -1,244 +1,699 @@
 # Configuration file
 
 client {
+    # Align HUD with left (if true) or right (if false)
+    B:AlignHUDLeft=true
+
+    # How much light to produce if ambient lighting is enabled.
     I:AmbientLightingLevel=15
+
+    # Allow sneak+scroll to change Configurator modes.
     B:ConfiguratorModeScroll=true
+
+    # Should active machines produce block light.
     B:EnableAmbientLighting=false
```

Comment:
Personally I think this can be set back to true as it is no longer a source of massive lag in 9.7.0

Reply:
I'm going to leave this one disabled for the time being in case it was disabled due to an unfavourable interaction with shaders or another mod.

Reply:
IIRC that was during one of the few streams of darkosto I actually watched and it was due to people informing him it caused lag (which it did until the overhauls done in mekanica that are part of mekanism 9.7.0). Though it is fair to leave it be until someone else decides to re-enable it.

Reply:
I feel we should leave it off for a few iterations of Mekanism just to be sure there isn't going to be an issue. By the time the update is ready, this may as well be good to enable, but we'll leave it for now.

Reply:
That actually reminds me that the forge config for `allowEmissiveItems` can be enabled again as it was disabled at me pointing out to necron that it made some mekanism items invisible (which no longer happens). Also @sk2048 I would recommend at the very least updating it to 9.7.1 from the 9.7.0 (if not just waiting to 9.7.2) as 9.7.0 has a couple crashes that 9.7.1 fixed